File: java/2.x/src/com/relevantcodes/extentreports/source/ScreencastHtml.java
package com.relevantcodes.extentreports.source;

public class ScreencastHtml {
    public static String getSource(String screencastPath) {
        return "<video id='video' width='50%' controls>" +
        			"<source src='file://" + screencastPath +"'>" +
        			"Your browser does not support the video tag." + 
        		"</video>";
    }
}

File: dotnet/RelevantCodes.ExtentReports/RelevantCodes.ExtentReports/View/ScreencastHtml.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace RelevantCodes.ExtentReports.View
{
    internal class ScreencastHtml
    {
        public static string GetSource(string ScreencastPath)
        {
            return "<video id='video' width='50%' controls>" +
                    "<source src='file:///" + ScreencastPath + "'>" +
                    "Your browser does not support the video tag." +
                "</video>";
        }
    }
}
